packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,24 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let filterDotFiles = function (x) {
+    return x[0] === ".";
+  };
+  let copyFilesFilter = () => false;
+
   function handle(filename) {
     if (!fs.existsSync(filename)) return;
 
     let stat = fs.statSync(filename);
-
     if (stat.isDirectory(filename)) {
       let dirname = filename;
-
-      _.each(util.readdir(dirname), function (filename) {
+      if (commander.copyDotFiles) {
+        copyFilesFilter = filterDotFiles;
```

Comment:
No worries, the fault might completely be on my own misunderstanding :)

I was under the impression that the flag would enable the copy operation to _include_ dotfiles, not _just_ dotfiles.